be/src/formats/parquet/scalar_column_reader.h

```diff
@@ -145,6 +146,7 @@ class ScalarColumnReader final : public RawColumnReader {
     }
 
     void set_can_lazy_decode(bool can_lazy_decode) override {
+        _can_lazy_convert = can_lazy_decode;
```

Comment:
Only if a column is lazily decodable, it is lazily convertible?

Reply:
no, we can treat convert as a kind of decode, we set_can_lazy_decode when that column has no predicate.

Reply:
Well, the name of this method can be modified to make it more readable.

Reply:
However, I like the generalization concept of treating convert as a kind of decode

Reply:
OK, i think i can change the name _can_lazy_decode to _can_lazy_dict_decode, so dict_decode and convert are all belongs to decode.